### DISCUSSION TITLE
How to get back on the previous Vault and get all the previous notes?

### DISCUSSION THREAD

**Discussion by s-earnest** [dendronhq/dendron#1042] [Q&A]
Hello,
I have to say that I like Dendron. I tested Dendron for a one week, and I like it but I have a hard time to get back on the vault where I had my previous notes. The initialize workplace makes me to start over again from scratch. 
Any suggestion? 
Thank you so much for your help.

**Comment by SeriousBug** [comment]
Hi!

Please try opening the workspace file in your Dendron workspace.

![](https://i.imgur.com/Wy10M9x.png)

Then, you can click on the open workspace button.

![](https://i.imgur.com/iDeOx6V.png)

This should bring you back to your workspace. In the future, you can also open the workspace from "recent workspaces". Make sure to open the one that says "(Workspace)".

![](https://i.imgur.com/O1PO6Lz.png)

Please let me know if this helps.
